jvm-libs/linea/core/long-running-service/src/test/kotlin/net/consensys/zkevm/LineaPeriodicPollingServiceTest.kt

```diff
@@ -102,7 +148,13 @@ class PeriodicPollingServiceTest {
         result
       }
     }
-    val pollingService = PollingService(vertx, pollingInterval, log, mockAction = action)
+    val pollingService = PollingService(
```

Comment:
Not related to this PR, but I dun see what's the intention of line:144-148? do u think we can simplify it into simply :
```
asyncDelay(vertx, 5.milliseconds)
```